App.svelte

```diff
@@ -39,6 +43,21 @@
     showInstructions = !showInstructions;
     showAbout = false;
   }
+
+  let boundaryGeojson;
```

Comment:
I lifted this out of `BoundaryLayer`, so we can also pass it down to `ZoomOutMap`. Right now there's not very much global app state plumbed around here; we will definitely need to think about organizing once it grows more.